docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/snowplow-ecommerce/index.md

```diff
@@ -0,0 +1,368 @@
+---
+title: "Snowplow Ecommerce"
+sidebar_position: 16000
```

Comment:
Why not 100000000? Seriously though it’s funny to see these numbers growing and growing